x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -183,12 +160,12 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         esArchiver.load('x-pack/test/functional/es_archives/infra/alerts'),
         esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
         esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_hosts_processes'),
-        kibanaServer.savedObjects.cleanStandardList(),
```

Comment:
Can you help me understand why this is not needed anymore?

Reply:
I removed it because unlike other tests that have this, there are no saved objects being created that I can see.  I guess I couldn't figure out why it's there to begin with and since no tests are breaking I'm assuming it's not needed.  I removed it because if we delete all the saved objects before I am able to uninstall the package, after the tests have run, I won't be able to uninstall the package's assets because it depends on the saved object (`epm-packages`).